genestack_client/genestack_exceptions.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, current_version, required_version=None):
         :type required_version: distutils.version.StrictVersion
         """
         if required_version:
-            branch = 'master' if required_version.prerelease else 'stable'
+            branch = 'inception_master' if required_version.prerelease else 'inception_stable'
```

Comment:
Don't forget to adjust target branch for this PR.
I'd recommend creating those branches (matching `master` and `stable` for the moment) right now.

Reply:
I resolve some comment in separate branch from master, will create new branch before merge.